.github/workflows/pmm-ui-tests-matrix-upgrade.yml

```diff
@@ -1,85 +1,153 @@
-name: PMM2 UI Tests Upgrade Matrix
+name: PMM Upgrade UI Tests
 
 on:
+  schedule:
+    - cron: '0 0 * * *'
   workflow_dispatch:
     inputs:
       pmm_ui_tests_branch:
-        description: 'Target branch for pmm-ui-tests repository'
+        description: 'pmm-ui-tests repository branch'
         default: 'main'
         required: true
-      pmm_server_version:
-        description: 'Version of the pmm server used for testing'
-        default: 'dev-latest'
-        required: true
-      pmm_server_docker_tag:
-        description: 'Use when tag different to: "perconalab/pmm-server" is required'
-        type: string
-      pmm_client_version:
-        description: 'Version of the pmm client used for testing'
-        default: 'dev-latest'
-        required: true
-      pmm_client_upgrade_version:
-        description: 'Version of the pmm client to upgrade to, if left empty, then version based on repository will be selected.'
-        type: string
       pmm_qa_branch:
-        description: 'Branch for the pmm-qa repository.'
+        description: 'pmm-qa repository branch(for setup)'
         default: 'main'
         required: true
+      pmm_server_start_version:
+        description: 'PMM Server version to upgrade (latest|dev-latest|x.xx.x|x.xx.x-rc)'
+        default: 'latest'
+        required: true
```

Comment:
I would still prefer docker tag here. We might need a different tag from. Keep this just add option for custom tag.

Reply:
Sure, let's evaluate it. I suggest to create a poll and ask team how many times and how often they used docker image form repo other then: `percona/pmm-server` and `perconalab/pmm-server`. The reason is I'm not aware that it's even exist.